external/storm-hdfs/src/test/resources/log4j.properties

```diff
@@ -20,7 +20,4 @@ log4j.rootLogger = WARN, out
 
 log4j.appender.out = org.apache.log4j.ConsoleAppender
 log4j.appender.out.layout = org.apache.log4j.PatternLayout
-log4j.appender.out.layout.ConversionPattern = %d (%t) [%p - %l] %m%n
-
-log4j.logger.org.apache.storm.hdfs = INFO
```

Comment:
I don't think we're piping Storm logs to log4j 1, so I removed this and added a Log4j 2 file instead.